tests/acceptance/30_generic_methods/permissions_dirs_recursive.cf

```diff
@@ -50,6 +79,7 @@ bundle agent test
 {
   methods:
     "ph1" usebundle => permissions_dirs_recursive("${init.directory}", "${init.mode}", "${init.owner}", "${init.group}");
+    "ph2" usebundle => permissions_dirs_recursive("${init.directory2}", "${init.mode2}", "", "${init.group2}");
```

Comment:
Could you change the test to remove the mode and only change the owner or group?
Or add a new one.
The current one changes 2 things at the same time and make it unclear on its behaviour

Reply:
ok